response_to_reviewers.tex

```diff
@@ -0,0 +1,117 @@
+\documentclass[a4]{article}
+
+\usepackage{hyperref}
+
+\title{Response to reviews}
+
+\begin{document}
+
+\maketitle
+
+\section{Comments from editor}
+
+Thank you for the review of our paper. We are delighted by the reviews: the
+second author feeling there are no modifications necessary and the second
+pointing out clarifications which have given us the opportunity to improve the
+manuscript.
+
+\begin{quote}
+	1. Please ensure that your manuscript meets PLOS ONE's style requirements,
+	including those for file naming. The PLOS ONE style templates can be found at
+	\url{http://www.journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf}
+	and
+	\url{http://www.journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf}
+\end{quote}
+
+\begin{quote}
+    2. Please remove your figures from within your manuscript file, leaving only
+    the individual TIFF/EPS image files, uploaded separately.  These will be
+    automatically included in the reviewers’ PDF.
+\end{quote}
+
+All files have been renamed as requested and otherwise, the document follows the
+given LaTeX style template.
+
+\section{Comments from first referee}
+
+\begin{quote}
+    1) The authors discussed about as many as 164 strategies, which were
+    trained/generated via different algorithms. Since understanding how the
+    strategies work plays an important role in understanding why some of them
+    succeed, I strongly suggest the authors to introduce the strategies in a more
+    detailed and clear way, especially those based on finite state machines as I
+    find no reference of them yet some of them perform well in the comparisons.
+\end{quote}
+
+This is a well made point by the referee: ensuring the reader can understand
+these is vital to the paper.
+We have clarified the description of the strategies:
+
+\begin{itemize}
+    \item Highlight that the appendix contains references for each strategy
+        including pointing to a recently published paper that goes in to details
+        about the structure of these strategies.
+    \item A more formal definition of a finite state machine explaining how they
```

Comment:
Let's make sure we have the appropriate Ashlock et al references since they go into much more detail.